parser/CMakeLists.txt

```
@@ -129,3 +129,6 @@ new_ec_test(empty_editorconfig_file empty.in test4 "^[ \t\n\r]*$")
 new_ec_test(newlines_only_editorconfig_file newlines_only.in test4 "^[ \t\n\r]*$")
 new_ec_test(comments_only_editorconfig_file comments_only.in test4 "^[ \t\n\r]*$")
 new_ec_test(comments_and_newlines_editorconfig_file comments_and_newlines.in test4 "^[ \t\n\r]*$")
+
+# A key set to empty string
+new_ec_test(key_set_to_empty_string empty_values.in test1 "^key1=[\n\r]*key2=[\n\r]*key3=[\n\r]*$")
```

I think this should be
```suggestion
new_ec_test(key_set_to_empty_string empty_values.in test1 "^key1=[\n\r]+key2=[\n\r]+key3=[\n\r]+$")
```
so we don't accept, e.g., `key1=key2` as passing this test.

---

Thanks fixed in 9877f04